hdmi/interfaces/hdmi_interface.py

```diff
@@ -0,0 +1,69 @@
+from myhdl import Signal
+
+
+class HDMIInterface:
+
+    def __init__(self, clock):
+
+        """
+         This interface is the internal interface modeled after
+         the xapp495 external HDMI interface
+        """
```

Comment:
I would add more information here on the HDMI link.  Describe each signal and describe what is required to send a frame or video stream.